scania-digital-design-system/tegel
# chore(tooltip): use singular Capital Case where possible

**Describe pull-request**  
Updated comments and docs to use singular capital case.

**Solving issue**  
Part of: https://tegel.atlassian.net/browse/DTS-1329

**How to test**  
1. Go to storybook
2. Check in Components -> Tooltip
3. Make sure it is always referenced with singular capital case (Tooltip)
